src/main/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilder.java

```diff
@@ -31,17 +31,17 @@ public class CriteriaQueryBuilder {
 
   final SnapshotBuilderSettings snapshotBuilderSettings;
 
-  private record OccurrenceTable(String tableName, String idColumnName) {}
+  public record OccurrenceTable(String tableName, String idColumnName) {}
 
-  private static final Map<Integer, OccurrenceTable> DOMAIN_TO_OCCURRENCE_TABLE =
+  public static final Map<Integer, OccurrenceTable> DOMAIN_TO_OCCURRENCE_TABLE =
```

Comment:
This can remain `private` as it's only used in `getOccurrenceTableFromDomain()`
```suggestion
  private static final Map<Integer, OccurrenceTable> DOMAIN_TO_OCCURRENCE_TABLE =
```

Reply:
Got it!